osu.Game/Localisation/SkinEditorStrings.cs

```diff
@@ -42,7 +42,17 @@ public static class SkinEditorStrings
         /// <summary>
         /// "Currently editing"
         /// </summary>
-        public static LocalisableString CurrentlyEditing => new TranslatableString(getKey(@"currently_editing"), "Currently editing");
+        public static LocalisableString CurrentlyEditing => new TranslatableString(getKey(@"currently_editing"), @"Currently editing");
+
+        /// <summary>
+        /// "Revert to default."
+        /// </summary>
+        public static LocalisableString RevertToDefaultDescription => new TranslatableString(getKey(@"revert_to_default"), @"Revert to default.");
+
+        /// <summary>
+        /// "Return the skin to its default state"
+        /// </summary>
```

Comment:
This is all wrong. like all of it. description is header, dialog is dialogue, there's a full stop

Reply:
and like, there's already a string for rever to default